src/current/v23.1/security-reference/authorization.md

```diff
@@ -172,7 +172,7 @@ For more examples of default privileges, see the examples on the [`SHOW DEFAULT
 
 ## Role options
 
-Users' authorization to perform certain actions are governed not by grants but by [`role options`]({% link {{ page.version.version }}/create-user.md %}#role-options). These options govern whether users can perform actions such as:
+A user's authorization to perform certain actions is governed by [`role options`]({% link {{ page.version.version }}/create-user.md %}#role-options) rather than grants. For example:
```

Comment:
hm, this is beyond the initial scope of this PR, but the list below is inaccurate now.

As per https://www.cockroachlabs.com/docs/stable/security-reference/authorization#authorization-models: 
> any legacy role options that now have corresponding system-level privilege versions are deprecated in CockroachDB v22.2, though both are supported alongside each other

- "pausing, resuming, and canceling jobs" now has a CONTROLJOB system privilege. 
- "Creating or renaming databases" now has a CREATEDB system privilege.
- "Managing authentication for other users" now has CREATEROLE and CREATELOGIN system privileges.
- "Modifying cluster settings" now has MODIFYCLUSTERSETTING and MODIFYSQLCLUSTERSETTING system privilleges.
- "Creating changefeeds" now has a CHANGEFEED system privilege.

These are documented at: https://www.cockroachlabs.com/docs/stable/security-reference/authorization#supported-privileges

Maybe we could actually remove this whole section?

Reply:
Good idea, let me take a look. It's not ideal to have the content duplicated in multiple places (without using an include to do it).